### PR TITLE
OSSMDOC-557: update supported config topic

### PR DESCRIPTION
**Pull request by rh-tokeefe**
- Aligned team: Dev Tools
- For branches: 4.6 - 4.11
- Jira: https://issues.redhat.com/browse/OSSMDOC-557
- Direct link to doc preview: https://deploy-preview-44836--osdocs.netlify.app/openshift-enterprise/latest/service_mesh/v2x/preparing-ossm-installation.html#ossm-supported-configurations_preparing-ossm-installation
- SME review: @ tbd
- QE review: @ tbd
- Peer review: jstickler

### CODE REVIEW

File: modules/ossm-supported-configurations.adoc [hunk 1]
```diff
@@ -9,7 +9,7 @@
 
 The following configurations are supported for the current release of {SMProductName}:
 
-* Red Hat {product-title} version 4.x.
+* Red Hat {product-title} version 4.9.
```
Comment (JStickler) [inline modules/ossm-supported-configurations.adoc:12]: Let's make this "version 4.9 or later."

File: modules/ossm-supported-configurations.adoc [hunk 2]
```diff
@@ -45,8 +45,7 @@ For additional information about {SMProductName} lifecycle and supported configu
...
 * Jaeger agent as a sidecar is the only supported configuration for Jaeger. Jaeger as a daemonset is not supported for multitenant installations or OpenShift Dedicated.
 
-[id="ossm-supported-configurations-adapters_{context}"]
-== Supported Mixer adapters
+[id="ossm-supported-configurations-webassembly_{context}"]
+== Supported WebAssembly module
 
-* This release only supports the following Mixer adapter:
-** 3scale Istio Adapter
+* 3scale WebAssembly is the only supported WebAssembly module.
```
Comment (JStickler) [inline modules/ossm-supported-configurations.adoc:51]: "3scale WebAssembly is the only **supported** WebAssembly module."  I could see people taking this literally, let's change this to "3scale WebAssembly is the only **provided** WebAssembly module.  You can create custom WebAssembly modules."

**Review by JStickler — COMMENTED** [review]
Just a couple of small changes.

File: modules/ossm-supported-configurations.adoc (hunk 1 above)
Comment (rh-tokeefe) [reply modules/ossm-supported-configurations.adoc:12]: Done

File: modules/ossm-supported-configurations.adoc (hunk 2 above)
Comment (rh-tokeefe) [reply modules/ossm-supported-configurations.adoc:51]: Done

**Review by JStickler — APPROVED** [review]
LGTM.  Remove the [WIP] and we're ready to merge.

**Pull request merged**